blackbox/docs/sql/statements/set-license.rst

```diff
@@ -0,0 +1,60 @@
+.. highlight:: psql
+.. _ref-set-license:
+
+===============
+``SET LICENSE``
+===============
+
+Sets the license Key.
+
+.. rubric:: Table of Contents
+
+.. contents::
+    :local:
+
+Synopsis
+========
+
+::
+
+    SET LICENSE license_key
+
+Description
+===========
+
+``SET LICENSE`` registers a CrateDB license key to the cluster.
+If the license registration is successful, CrateDB will continue to operate
+in `Enterprise Edition`_ until the license expiry date.
+
+.. NOTE::
```

Comment:
I think this note should just link to the "enterprise features" where we describe the expiration/switching to community edition behaviour already.

Reply:
eg. For more details on switching between Enterprise and Community edition and handling expired licensese see :ref:`enterprise_features` 

Or something along those lines.